Comments/ClientApp/src/components/CommentList/Drawer.scss

```diff
@@ -41,13 +38,18 @@
     &:hover {
       cursor: pointer;
     }
-    &--comments,&--questions {
+    &--comments,
+    &--questions {
       color: $colour-nice-white;
       box-shadow: inset -10px 0 10px rgba(0, 0, 0, 0.3);
     }
-    &--comments.active,&--questions.active{
+    &--comments.active,
+    &--questions.active {
       background: $colour-btn-cta;
     }
+    &-text {
+      font-size: 0.9rem;
```

Comment:
It probably doesn't, but does this cause any issues with colour contrast? I know some of our colours only pass when at full size text.

Reply:
Doesn't appear to be a problem when I've checked it.